CMakeLists.txt

```diff
@@ -625,14 +625,18 @@ if(FFMPEG_FOUND)
   )
 endif(FFMPEG_FOUND)
 
+if(DEFINED ENV{LIB_FUZZING_ENGINE})
```

Comment:
This should move down to after `if(WITH_EXAMPLES)` as it's in the examples.

Well... actually it isn't an example but a fuzzing driver. So I see two "Right Way" approaches here:

a) it should move to tests but then also be integrated with https://github.com/LibVNC/libvncserver/blob/master/.github/workflows/ci.yml (i.e. be part of the test suite)
b) If that's too cumbersome, maybe this driver thingy should not be part of LibVNCServer at all and instead a FUZZING.md be added to the repo root explaining to other users how to use the instrumentation added in this PR.

Reply:
I tried the first option in the latest branch version.

How do you like it ?
I just build it and do not run it.
If you want to run it, you have to add an argument because by default fuzzing will run until it triggers a bug...
And if you want to do that, I recommend you use CIFuzz GitHub actions (which works only if project is integrated into oss-fuzz).

For information, I have seen different projects having fuzz being parts of either examples, tests, or in a stand alone fuzz directory...